ZengineHQ/mayan
# show response if error is not parsable;

To test 

cd ~/projects/mayan   # go into the mayan checkout
npm link                    # creates global link
cd ~/projects/zn-plugin-backend-plugin  # go into some other package directory.
npm link mayan              # link-install the package

Then you can run your mayan deployment like normal to see the new errors
